### ISSUE TITLE
Segmentation fault when auto-tagging

### ISSUE THREAD

**Issue by Turiok** [comictagger/comictagger#572]
Really often, When I try to auto-tag my CBR or CBZ, A segmentation fault appear.
I left the default option for auto-tag.
I would like to enable more debug information to help.
But I don't know how to do that.

Version :  comictagger_1.6.0-alpha8
OS : Windows 11
Log : 
`Examining covers for  ID: 139699 Michel Vaillant - Nouvelle saison (2012) ...Segmentation fault`
Metadata Source : "Comic Vine"
Read style : "ComicRack"
Modify style : "ComicRack"
I configure my API comic vine key.
And Give a path for the rar.exe

**Comment by lordwelch** [comment]
Please provide the full log it should be located at `C:\Users\<username>\AppData\Local\ComicTagger\log`

**Comment by Turiok** [comment]
Here the log : 
<details>
  <summary>Click me</summary>

```
2023-11-25T12:30:36 | comictagger | INFO | ComicTagger Version: 1.6.0a8 running on: Windows PyInstaller: Yes
2023-11-25T12:30:36 | comictagger | DEBUG | user_data_dir: C:\Users\<my_user>\AppData\Local\ComicTagger\ComicTagger
2023-11-25T12:30:36 | comictagger | DEBUG | user_config_dir: C:\Users\<my_user>\AppData\Local\ComicTagger\ComicTagger
2023-11-25T12:30:36 | comictagger | DEBUG | user_cache_dir: C:\Users\<my_user>\AppData\Local\ComicTagger\ComicTagger\Cache
2023-11-25T12:30:36 | comictagger | DEBUG | user_state_dir: C:\Users\<my_user>\AppData\Local\ComicTagger\ComicTagger
2023-11-25T12:30:36 | comictagger | DEBUG | user_log_dir: C:\Users\<my_user>\AppData\Local\ComicTagger\ComicTagger\Logs
2023-11-25T12:30:37 | comictagger | DEBUG | Installed Packages
2023-11-25T12:30:37 | comictagger | DEBUG | comictagger	1.6.0a8
2023-11-25T12:30:37 | comictagger | DEBUG | importlib-metadata	6.8.0
2023-11-25T12:30:37 | comictagger | DEBUG | metron-talker	0.1.2
2023-11-25T12:30:37 | comictagger | DEBUG | pycountry	22.3.5
2023-11-25T12:30:56 | comictalker.talkers.comicvine | INFO | Comic Vine searching: michel vaillant t
2023-11-25T12:30:56 | comictaggerlib.seriesselectionwindow | INFO | Length: 0, 6, 94
2023-11-25T12:31:00 | comictaggerlib.issueselectionwindow | INFO | successfully loaded QWebEngineView
2023-11-25T12:31:26 | comictalker.talkers.comicvine | INFO | Comic Vine searching: michel vaillant t
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant ; ratio: 93 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant ; ratio: 93 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant  ; ratio: 93 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant ; ratio: 93 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant ; ratio: 93 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant - Intégrale ; ratio: 80 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant - Histoires courtes ; ratio: 68 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant: Legendary Races ; ratio: 66 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant - Nouvelle saison ; ratio: 66 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Vaillant Short Story Classics ; ratio: 64 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Michel Ange ; ratio: 64 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T&A ; ratio: 11 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Mr. T and the T-Force ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: S.T.A.T. ; ratio: 36 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Gruppo T.N.T. ; ratio: 27 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: SH-T ; ratio: 28 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T-Dragon ; ratio: 24 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: ティードラゴン ; ratio: 0 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T－DRAGON ; ratio: 24 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Mr. T ; ratio: 28 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Mr. T ; ratio: 28 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: ORBiT ; ratio: 18 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.E.S.T. Kitchen Infinity Comic ; ratio: 38 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Mr. T und die T-Force ; ratio: 32 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.V. Picture Stories ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.s ; ratio: 38 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.V. Strip ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.E.S.T ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: G.O.T.H. ; ratio: 25 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Chicken Sh!t ; ratio: 41 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Dread-noT ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: ドレッドノット ; ratio: 0 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Zo is 't leven! ; ratio: 22 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: The Adventures of Martial T. Holiday ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Just T & Cakes ; ratio: 20 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.V. Mundial ; ratio: 28 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: S.T.A.R. Corps ; ratio: 21 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Collection T. V. Pocket  ; ratio: 36 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Casper T.V. Showtime ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.E.S.T. ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: U.T.F. ; ratio: 27 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Dog T.A.G.S. ; ratio: 23 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Retief of the C.D.T. ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Best of H.T. Webster ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Free S**t ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Free Shit ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S. ; ratio: 38 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Shiritsu T Joshi Gakuen ; ratio: 25 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: 私立T女子学園 ; ratio: 8 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.E.S.T. ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: G.I.L.T. ; ratio: 41 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.H.E. CAT ; ratio: 38 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Infini-T Force ; ratio: 32 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: E.T.O. ; ratio: 36 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Kirika C.A.T.s ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: C.A.T.s: Critical Advanced Tactical Suit ; ratio: 37 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: キリカC.A.T.s ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T-Bird & Throttle ; ratio: 25 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T-REX na Kanojo ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T-REXな彼女 ; ratio: 16 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: My Girlfriend is a T-Rex ; ratio: 38 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: She's a T-Rex ; ratio: 37 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Mr. T Graphic Novel ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.J.Hooker Annual ; ratio: 29 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T-Minus: The Race to the Moon ; ratio: 37 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Mashōnen B.T. ; ratio: 41 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: 魔少年ビーティー ; ratio: 0 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T&A: Before & After ; ratio: 25 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Brain Boy: The Men From G.E.S.T.A.L.T. ; ratio: 25 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.E.S.T. Kitchen Halloween Special Infinity Comic ; ratio: 36 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.E.S.T. Kitchen Holiday Special Infinity Comic ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T.E.S.T. Kitchen Thanksgiving Special Infinity Comic ; ratio: 32 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: FISK the S.U.B.S.T.I.T.U.T.E. ; ratio: 29 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Brain Boy: The Men From G.E.S.T.A.L.T. ; ratio: 25 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: FISK the S.U.B.S.T.I.T.U.T.E. ; ratio: 29 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: S.A.T.A.N. Picture Library ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Spies Anonymous To All Nations Picture Library ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S Sourcebook ; ratio: 27 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: My Girlfriend is a T-Rex ; ratio: 38 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S.: Mosaic ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: K.R.I.X.T. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: T. Perran Mitchell's Short Trips ; ratio: 42 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S Trilogy ; ratio: 35 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Nextwave: Agents of H.A.T.E. ; ratio: 36 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.s Adventures ; ratio: 32 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S Trilogy ; ratio: 35 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Alterna GIANTS: T-Bird & Throttle ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.s Annual ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.s: Homecoming ; ratio: 32 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: The Tremendous Trump: World War T ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.I.T.C.H. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Ведьма ; ratio: 0 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: The Manly Adventures of Henry T. Manchester ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Spawn/WildC.A.T.S ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S Special ; ratio: 29 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.s/Aliens ; ratio: 36 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: JLA/WildC.A.T.s ; ratio: 40 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.S: Compendium ; ratio: 27 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: S.O.F.T. Corps ; ratio: 19 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Spawn/WildC.A.T.S. ; ratio: 31 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Don’t Come in Here ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Sh*t My President Says ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: Shit My President Says ; ratio: 30 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: With a Little T.L.C. ; ratio: 34 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.i.t.c.h. (As Bruxinhas) ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.T.s: Covert Action Teams ; ratio: 39 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildC.A.Ts: Covert Action Teams ; ratio: 40 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: WildCATs: Covert Action Teams ; ratio: 40 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: W.I.T.C.H.: The Graphic Novel ; ratio: 35 ; match threshold: 91
2023-11-25T12:31:26 | comicapi.utils | DEBUG | search title: Michel Vaillant - T ; record title: I.T.: The Secret World of Modern Banking ; ratio: 33 ; match threshold: 91
2023-11-25T12:31:27 | comictaggerlib.issueidentifier | DEBUG | Width: 1280 Height: 1737, ratio: 0  0 ratio met: False
```
</details>

No errors shown

**Comment by lordwelch** [comment]
So I happened on this crash when working on #579, which I was working on in order to make sure more info was actually in the logs so I could find out what this issue was....

Anyways this should only happen if you have opened up the settings window before using auto-tag. So as a work around you have to restart ComicTagger after modifying any settings.

**Comment by lordwelch** [comment]
Fixed in 5b1c92e7b8b5becf40194141ee94d61b1cccb771
